docs/pages/example/geojson-layer-in-stack.html

```diff
@@ -9,6 +9,13 @@
 
 map.on('load', function () {
 
+    // Find the index of the bottom-most type: symbol layer in the map style
+    // Insert the layer beneath any symbol layer.
```

Comment:
this comment should be included after the comment on L31-36. Something like "in this case, we are inserting the new layer immediately before the first symbol layer in the style"